swap_meet/clothing.py

```diff
@@ -1,2 +1,13 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, fabric="Unknown", condition=0):
+        super().__init__(id, condition)    
+        self.fabric = fabric
+
+    def get_category(self):
+        return "Clothing"
```

Comment:
As I mentioned in the Item class, if you use the `self.__cls__.__name__` syntax, you don't even need to overload the `get_category()` method here!